.github/ISSUE_TEMPLATE/app_rendering_bug_report.yaml

```diff
@@ -0,0 +1,62 @@
+name: Application Card Rendering Bug Report
+description: Create a bug report for problems rendering Adaptive Cards in an Application
+title: "[Rendering] Bug Title Here"
+labels: [Area-Renderers, Bug, Triage-Needed]
+body:
+  - type: dropdown
+    validations:
+      required: true
+    attributes:
+      label: Target Platforms
+      description: Select the platform(s) that you are rendering on
+      multiple: true
+      options:
+        - NodeJS
+        - iOS
+        - Android
+        - WPF
+        - UWP
+  - type: input
+    validations:
+      required: true
+    attributes:
+      label: SDK Version
+      description: What version of the SDK are you using
+  - type: input
+    validations:
+      required: true
+    attributes:
+      label: Application Name
+      description: The name of the application that is having problems with Card Rendering
+      placeholder: Microsoft Teams
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Problem Description
+      description: Please enter a description of the issue. If you just have a question, please post [on StackOverflow instead](https://stackoverflow.com/questions/tagged/adaptive-cards)
```

Comment:
somewhat unrelated to this change - should we close down our GitHub discussions tab? if not, do we want to direct people there?

Reply:
Great idea - let's direct people there instead